contrib/cirrus/setup_environment.sh

```diff
@@ -142,6 +142,25 @@ case "$CI_DESIRED_NETWORK" in
     *)          die_unknown CI_DESIRED_NETWORK ;;
 esac
 
+# Database: force SQLite or BoltDB as requested in .cirrus.yml.
+# If unset, will default to BoltDB.
+case "$CI_DESIRED_DATABASE" in
+    sqlite)
+	warn "Forcing PODMAN_DB=sqlite"
+	echo "PODMAN_DB=sqlite" >> /etc/ci_environment
+	;;
+    boltdb)
+	warn "Forcing PODMAN_DB=boltdb"
+	echo "PODMAN_DB=boltdb" >> /etc/ci_environment
```

Comment:
Please add `PODMAN_DB` to the `PASSTHROUGH_ENV_EXACT` regex in `lib.sh` (otherwise rootless and containerized tasks will not receive it).

Reply:
Done